src/Illuminate/Foundation/Console/EnvironmentDecryptCommand.php

```diff
@@ -62,6 +62,10 @@ public function handle()
     {
         $key = $this->option('key') ?: Env::get('LARAVEL_ENV_ENCRYPTION_KEY');
 
+        if (! $key) {
+            $key = $this->ask('Decryption key');
```

Comment:
Maybe better to use https://laravel.com/docs/11.x/prompts#password

Reply:
Great idea, have changed this now